app/Providers/AppServiceProvider.php

```diff
@@ -11,18 +15,55 @@ class AppServiceProvider extends ServiceProvider
      *
      * @return void
      */
-    public function boot()
+    public function boot(): void
     {
-        //
+        $this->configFiles();
     }
 
     /**
      * Register any application services.
      *
      * @return void
      */
-    public function register()
+    public function register(): void
     {
-        //
+        $this->app->singleton(CustomizeConfigRuntime::class, CustomizeConfigRuntime::class);
+
+        $this->app->singleton(OptionManager::class, OptionManager::class);
+        $this->app->alias(OptionManager::class, 'options');
+        $loader = AliasLoader::getInstance();
```

Comment:
What's this for?

Reply:
Yah, the easiest way is just to use [Real-time Facades](https://mattstauffer.com/blog/real-time-automatic-facades-in-laravel-5-4/) when you do use OptionManager.